backend-rust/migrations/0001_initialize.up.sql

```diff
@@ -597,11 +600,17 @@ CREATE TABLE account_tokens (
         NUMERIC
         NOT NULL
         DEFAULT 0,
+    -- Every time an `account_token` is inserted or updated in this table, a sequential index is assigned 
+    -- to the operation and tracked in this sequence.
+    -- This sequence is used to sort/filter the newest tokens transferred to an account address.
+    change_seq BIGINT DEFAULT nextval('account_tokens_update_seq'),
 
     -- Ensure that each token_index and account_index pair is unique.
     CONSTRAINT unique_token_account_relationship UNIQUE (token_index, account_index)
 );
 
+CREATE INDEX non_zero_account_token_idx ON account_tokens (account_index) WHERE balance != 0;
```

Comment:
We need it to include the `change_seq` as well
```suggestion
CREATE INDEX non_zero_account_token_idx ON account_tokens (account_index, change_seq) WHERE balance != 0;
```